src/App.css

```diff
@@ -0,0 +1,132 @@
+/* Root variables */
+:root {
+  --primary: #FF8E3E;
+  --secondary: #21614A;
+  --tertiary: #7BC1DC;
+  --accent: #78D3AC;
+  --dark: #05041D;
+  --background: #f9fafc;
+  --grey: #D9D9DF;
+  --greyMedium: #B4B4BF;
+  --greyDark: #868696;
+}
+
+/* Global styles */
+html {
+  height: 100vh;
+  font-size: 0.85vw;
+}
+
+body {
+  height: 100%;
+  margin: 0;
+  font-family: 'Montserrat', -apple-system, BlinkMacSystemFont, 'Segoe UI',
+    'Roboto', 'Oxygen', 'Ubuntu', 'sans-serif';
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  color: #333333;
+  background-color: var(--background);
+  overflow: hidden;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+#root {
+  height: 100%;
+  width: 100%;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+a {
+  margin: 0 !important;
```

Comment:
Is using the "!important" tag a good practice?

Reply:
Actually curious

Reply:
Rather not use it, I do use it for some things like this a element because these have inherited styles from the default web style sheet. I have sometimes experienced that these elements already have an important styling which can not be overridden by default styling. This is for example also true for the font sizes I declared (fs1, fs2, etc.), because elements like h1 and h2 inherit their own important font sizes. Best practice would be to use those, but since we have our own little style guide, I change them with the custom styling using the important tag. Could just use p elements, but headers like h1 have an influence on metadata that is being collected by search aggregators like Google, so rather use some of them to indicate for example: the title in the header bar or a big tittle on the homepage which holds a powerful but short description of the Marketplace. Hope this all makes sense